runtime/crab/src/pallets/fee_market.rs

```diff
@@ -0,0 +1,57 @@
+// --- core ---
+use core::cmp;
+// --- substrate ---
+use frame_support::{traits::LockIdentifier, PalletId};
+use sp_runtime::{traits::UniqueSaturatedInto, Permill};
+// --- darwinia ---
+use crate::*;
+use darwinia_fee_market::{Config, RingBalance, Slasher};
+
+frame_support::parameter_types! {
+	pub const FeeMarketPalletId: PalletId = PalletId(*b"da/feemk");
+	pub const TreasuryPalletId: PalletId = PalletId(*b"da/trsry");
+	pub const FeeMarketLockId: LockIdentifier = *b"da/feelf";
+
+	pub const AssignedRelayersNumber: u64 = 3;
+	pub const MinimumRelayFee: Balance = 15 * COIN;
+	pub const Slot: BlockNumber = 50;
+	pub const CollateralPerOrder: Balance = 100 * COIN;
```

Comment:
Request review @AsceticBear

Reply:
Lower this for the same reason (unstable bridger), increase this in later runtimes.
```suggestion
	pub const AssignedRelayersNumber: u64 = 3;
	pub const MinimumRelayFee: Balance = 15 * COIN;
	pub const Slot: BlockNumber = 600;
	pub const CollateralPerOrder: Balance = 50 * COIN;
```